packages/row/src/react/index.js

```diff
@@ -91,16 +97,15 @@ const ActionBar = props => (
   <div {...styles.actionBar(props)} {...filterReactProps(props)} />
 )
 
-const ActionBarAction = props => {
+const ActionBarAction = withTheme(({ icon, ...props }) => {
```

Comment:
Why destructure out `icon` out?  Seems like `filterReactProps` would have taken it out, no?

Reply:
can't remember. i'll double check

Reply:
turns out it is a html attr -> https://github.com/jackyho112/react-html-attributes/blob/master/src/react-html-attributes.json#L37

so i guess that's while i'm filtering it